ui/job-view/details/tabs/PerformanceTab.jsx

```diff
@@ -16,6 +86,63 @@ export default class PerformanceTab extends React.PureComponent {
         role="region"
         aria-label="Performance"
       >
+        <div className="performance-panel-actions">
+          {
+            // If there is a profiler link, show this first. This is most likely
+            // the primary action of the user here.
+            profilerLink
+          }
+          {
+            // Just to be safe, use the same isPerfTest check the other
+            // "Create Gecko Profile" button uses in the action menu.
+            isPerfTest(selectedJobFull) ? (
+              <button
```

Comment:
The reactstrap `Button` component is pretty useful because it already has necessary attributes baked in (type, role, etc).